src/Mocking/JSInterop/MissingMockJsRuntimeException.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Egil.RazorComponents.Testing
+{
+    /// <summary>
+    /// Exception use to indicate that a MockJsRuntime is required by a test
+    /// but was not provided.
+    /// </summary>
+    [SuppressMessage("Design", "CA1032:Implement standard exception constructors", Justification = "<Pending>")]
+    public class MissingMockJsRuntimeException : Exception
+    {
+        /// <summary>
+        /// Identifer string used in the JSInvoke method.
+        /// </summary>
+        public string Identifier { get; }
+
+        /// <summary>
+        /// Arguments passed to the JSInvoke method.
+        /// </summary>
+        public object[] Args { get; }
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="MissingMockJsRuntimeException"/>
+        /// with the provided <see cref="Invocation"/> attached.
```

Comment:
```suggestion
        /// with the arguments used in the invocation.
```
`<see cref="Invocation"/>` is not a type as far as I remember, and we are not passing it in anyway.